lib/lightning_web/live/workflow_live/edit.ex

```diff
@@ -480,7 +480,7 @@ defmodule LightningWeb.WorkflowLive.Edit do
           |> Lightning.Repo.preload([
             :edges,
             triggers: Trigger.with_auth_methods_query(),
-            jobs: [:credential]
+            jobs: Workflows.jobs_ordered_subquery()
```

Comment:
I don't think we should be moving the preload of the Jobs credential into a function that given it's name says it's just for ordering.

Anyone using that function may or may not expect credentials to be preloaded.

I would suggest one of the following:

1) Setting the `:preload_order` on the `has_many` of the Workflow model. See [here](https://hexdocs.pm/ecto/Ecto.Schema.html#has_many/3-options)
2) Extend the returned query from `jobs_ordered_subquery` with a `preload(:credential)`, that will keep the credential preload out of that function.
3) Don't add a function, and have it all inline here. And we refactor/abstract these queries later.